rs/sns/testing/src/nns_dapp.rs

```diff
@@ -33,41 +25,41 @@ async fn validate_subnet_setup(pocket_ic: &PocketIc) {
     assert!(!app_subnet_ids.is_empty(), "No application subnets found");
 }
 
-async fn check_canister_exists(pocket_ic: &PocketIc, canister_id: &CanisterId) -> bool {
-    pocket_ic
-        .canister_info(*canister_id)
-        .await
-        .map(|_| true)
-        .unwrap_or(false)
-}
-
-pub async fn bootstrap_nns(pocket_ic: &PocketIc) {
+pub async fn bootstrap_nns(
+    pocket_ic: &PocketIc,
+    initial_mutations: Vec<RegistryAtomicMutateRequest>,
+    ledger_balances: Vec<(AccountIdentifier, Tokens)>,
+    neuron_hotkeys: Vec<PrincipalId>,
+) {
     // Ensure that all required subnets are present before proceeding to install NNS canisters
     // At the moment this check doesn't make a lot of sense since we are always creating the new PocketIC instance
     // with all the required subnets. However, in the future, we might want to be able to check externally provided
     // networks.
     validate_subnet_setup(pocket_ic).await;
-
     // Check if all NNS canisters are already installed
-    let canisters_exist = join_all(
+    let canisters_installed = join_all(
         ALL_NNS_CANISTER_IDS
             .iter()
-            .map(|canister_id| async { check_canister_exists(pocket_ic, canister_id).await }),
+            .map(|canister_id| async { check_canister_installed(pocket_ic, canister_id).await }),
     )
     .await;
 
-    if !canisters_exist.iter().any(|exists| *exists) {
+    if !canisters_installed.iter().any(|exists| *exists) {
```

Comment:
Should it say "installed" instead of "exists" here?

Reply:
Yes, because `canisters_installed` checks that canister _exists_ and is _installed_ (i.e. has non-empty WASM module)

Reply:
Sorry, I mean at the end of the line, where it says `any(|exists| *exists)` - that seems like it should have also gotten changed.  Kind of a nitpick :)